vassal-app/src/main/java/VASSAL/configure/ValidationReportDialog.java

```diff
@@ -38,6 +37,8 @@ public class ValidationReportDialog extends JDialog {
   private static final long serialVersionUID = 1L;
 
   private final CallBack callback;
+  JButton okButton;
+  JButton cancelButton;
```

Comment:
These are missing a `private`?

Reply:
Or, actually, why aren't they just local variables in the function below?

Reply:
Now they are! Whee!